editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -319,11 +319,12 @@ export function renderCoreElement(
       return <>{renderedChildren}</>
     }
     case 'JSX_TEXT_BLOCK': {
+      const unescaped = unescapeHTML(element.text)
       const parentPath = Utils.optionalMap(EP.parentPath, elementPath)
       if (parentPath == null || !EP.pathsEqual(parentPath, editedText)) {
-        return element.text
+        return unescaped
```

Comment:
this line rings an alarm bell to me. how would react render this text outside of utopia? we shouldn't be applying any special escaping that doesn't happen in create-react-app/codesandbox/etc

Reply:
That's a good point to which I don't have an obvious response 🤔 Should we maybe make a custom component that does the same unescaping?

Reply:
actually, it seems to be fine:

<img width="1109" alt="Screenshot 2022-12-14 at 2 02 40 PM" src="https://user-images.githubusercontent.com/1081051/207602210-61d36688-1dd5-4753-ae55-c0a9cabd5b8a.png">

Reply:
Was about to say, we should leave the text escaped 👍

Reply:
I believe this line should just be the original `return element.text` and that _should_ just work fine on the canvas. Right?

Reply:
unfortunately not 😬 

<img width="213" alt="Screenshot 2022-12-14 at 2 13 15 PM" src="https://user-images.githubusercontent.com/1081051/207604631-a43a24c0-11f9-4656-9442-b678360941f2.png">